scripts/buf_lint_check.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+jar_file_path=$?
+config_file_path=$?
+os_type=$?
+github_actions_path=$1
+
+check_protobuf() {
```

Comment:
Suggest either lint_protobuf_files or run_buf to be a bit clearer on what is specifically happening.

Reply:
done